src/actions/sso/consumeInteractionToken.ts

```diff
@@ -3,10 +3,11 @@ import { ThunkAction } from '../../store'
 import { ErrorCodes as LibErrorCode } from 'jolocom-lib/js/errors'
 import { interactionHandlers } from '..'
 
-
-export const consumeInteractionToken = (
-  jwt: string,
-): ThunkAction => async (dispatch, getState, sdk) => {
```

Comment:
We're using different names for the third argument throughout the actions: `sdk`, `backendMiddleware` and `agent`. I think we should use `agent` throughout the project for consistency, but also not sure if the refactor is worth it.